src/sqltest/parts/commandLine/commandLineService.test.ts

```diff
@@ -208,20 +214,23 @@ suite('commandLineService tests', () => {
 		connectionManagementService.setup(c => c.hasRegisteredServers()).returns(() => true).verifiable(TypeMoq.Times.atMostOnce());
 		connectionManagementService.setup(c => c.connectIfNotConnected(TypeMoq.It.isAny(), TypeMoq.It.isAny()))
 			.verifiable(TypeMoq.Times.never());
-		commandService.setup(c => c.executeCommand('mycommand'))
-			.returns(() => Promise.resolve())
+		let capturedArgs: any;
+		commandService.setup(c => c.executeCommand(TypeMoq.It.isAnyString(), undefined))
+			.returns((command, args) => {
+				capturedArgs = args;
+				return Promise.resolve();
+			})
 			.verifiable(TypeMoq.Times.once());
 		const configurationService = getConfigurationServiceMock(true);
 		let service = getCommandLineService(connectionManagementService.object, configurationService.object, capabilitiesService, commandService.object);
-		service.processCommandLine(args).then(() => {
-			connectionManagementService.verifyAll();
-			commandService.verifyAll();
-			done();
-		}, error => { assert.fail(error, null, 'processCommandLine rejected ' + error); done(); });
+		await service.processCommandLine(args);
+		connectionManagementService.verifyAll();
+		commandService.verifyAll();
+		should(capturedArgs).be.undefined();
```

Comment:
asserts that read like questions are weird.

Reply:
Agreed but that's the main Javascript verification framework for you 😄 I do like the general concept of fluid assertions but get where you're coming from on this.